app/models/protocol.rb

```diff
@@ -460,35 +460,11 @@ def any_service_requests_to_display?
     return self.service_requests.detect { |sr| !['first_draft'].include?(sr.status) }
   end
 
-  def has_per_patient_per_visit? current_request, portal
+  def has_line_items_of_type? current_request, portal, one_time_fee = false
```

Comment:
You do see a type, but it's one of two types, by definition, that is boolean in nature: one, or the other. Hence, boolean value. This also follows a convention seen elsewhere in the code, where a method having to do with services/line_items has a parameter of one time fee or not, as a boolean.